lib/Streaming/ByteString/Internal.hs

```diff
@@ -316,22 +312,39 @@ chunkOverhead = 2 * sizeOf (undefined :: Int)
 --             assert (l' <= l) $ return (B.PS fp 0 l', res)
 -- {-# INLINABLE packBytes' #-}
 
+-- | Convert a `Stream` of pure `Word8` into a chunked 'ByteStream'.
 packBytes :: Monad m => Stream (Of Word8) m r -> ByteStream m r
 packBytes cs0 = do
+  -- XXX: Why 32?  It seems like a rather small chunk size, wouldn't
+  -- smallChunkSize make a better choice?
```

Comment:
Weren't the chunks supposed to be 32 *kilobytes*?

Reply:
I don't know.  What I do know is that they were 32 bytes, for no obvious reason.

Reply:
Perhaps the issue is a concern around the efficiency of `toList` from `Streaming.Prelude`:
```
toList :: Monad m => Stream (Of a) m r -> m (Of [a] r)
toList = fold (\diff a ls -> diff (a: ls)) id (\diff -> diff [])
```
This builds a chain of "prepend" thunks, that are then applied to `[]`.  Is this a costly construction to run on a list of 4k elements? I'm not changing this, we can look into whether a larger number is warranted later...